packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -1209,15 +1209,13 @@ class _RenderTheater extends RenderBox with ContainerRenderObjectMixin<RenderBox
       assert(!child.debugNeedsLayout);
       final StackParentData childParentData = child.parentData! as StackParentData;
       double? candidate = child.getDistanceToActualBaseline(baseline);
-      if (candidate != null) {
-        candidate += childParentData.offset.dy;
-        if (result != null) {
-          result = math.min(result, candidate);
-        } else {
-          result = candidate;
-        }
+      candidate += childParentData.offset.dy;
+      if (result != null) {
+        result = math.min(result, candidate);
+      } else {
+        result = candidate;
       }
-      child = childParentData.nextSibling;
+          child = childParentData.nextSibling;
```

Comment:
Would you mind reverting this as well?

Reply:
> Would you mind reverting this as well?

Done, have no idea why my IDE did this, I disabled all the formatting options in my VS Code settings because the VS Code flutter project settings is not overriding my original for some reason, it might be because of some extension or its settings.